### PR TITLE
docs: remake new component docs

### LINKED ISSUE #823: Update `New Component` page
#### Is this a bug, enhancement, or feature request?
enhancement
#### Briefly describe your proposal.
Update the new component page
https://sap.github.io/fundamental-ngx/new-component

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #823

### CODE REVIEW

File: NEW_COMPONENT.md [hunk 1]
```diff
@@ -8,80 +8,159 @@ This repository contains two separate projects - the component library and the d
...
+```
+
+You should see 'Poster' appear in the side navigation under 'Components'.  Clicking the link will load a page that says `poster works!`
+
+
+## Add an image to the Poster component template
+
+In the library source, open `poster.component.html`, remove the default code and add an image with a placeholder.
+
+```<img src="https://placeimg.com/400/600/nature'"/>```
```
Comment (MattL75) [inline NEW_COMPONENT.md]: tag as html

File: NEW_COMPONENT.md [hunk 2]
```diff
@@ -8,80 +8,159 @@ This repository contains two separate projects - the component library and the d
...
+    ]
+},
+```
+You will need to import `PosterComponent` and `PosterDocsComponent` in this file as well.
+
+Next, open `documentation.component.ts` and add the following to the `components` array:
 
-![alt text](assets/images/new_component/finished-poster-component.png "Finished Poster Component")
+```javascript
+{ url: 'poster', name: 'Poster' }
```
Comment (MattL75) [inline NEW_COMPONENT.md]: I think json is an option for these code blocks

File: NEW_COMPONENT.md [hunk 3]
```diff
@@ -8,80 +8,159 @@ This repository contains two separate projects - the component library and the d
...
+Note that we're using raw-loader to import the poster example code as raw text.  This text will be rendered as the example source.
 
-```<img src="https://placeimg.com/400/600/nature'"/>```
+In `poster-docs.component.html`, we'll provide a brief explanation of the poster component, and we'll add the poster component itself, along with the code example.
+
+```
+<h2>Poster</h2>
+<description>
+    The Poster component displays a photograph taken from an image placeholder site.
+</description>
```
Comment (MattL75) [inline NEW_COMPONENT.md]: Tag this block as html

File: NEW_COMPONENT.md (hunk 2 above)
Comment (mikerodonnell89) [reply NEW_COMPONENT.md]: keys must be strings in json, which is not what we're using for the rest of these objects

**Pull request merged**